src/ioctls/vcpu.rs

```diff
@@ -1045,6 +1045,15 @@ impl VcpuFd {
         Ok(())
     }
 
+    /// Enables guest debugging and sets specific debug registers for x86
```

Comment:
Looking at the KVM documentation, it seems it is supported on: x86, s390, ppc, arm64. Can you update the description?

Can you add the arguments and example sections the documentation for this function?  You can check the documentation of other functions to get an inspiration of how it should look like.

Reply:
Updated the description